src/progressbar/styled-components.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+export const Root = styled('div', props => {
+  return {
+    width: '100%',
+  };
+});
+
+export const Bar = styled('div', props => {
+  const {$theme, $value, $successValue} = props;
+  const {colors, sizing, borders} = $theme;
+  let background = 'linear-gradient(to right';
+  const fillColor = colors.primary400;
+  const emptyColor = colors.mono400;
+  background += ', ';
+  background += fillColor;
+  background += ' ' + ($value / $successValue) * 100 + '%, ';
+  background += emptyColor;
+  background += ' 0% ';
+  background += ', ' + emptyColor + ' 0%';
+  background += ')';
```

Comment:
Using two elements may also make it easier to customize for customers – they can change the background color of the bar or backdrop independently without needing to do a linear gradient dance.

Reply:
do you think we need to? there was no requirement to have animation around it, it can jump from one progress to another pretty quickly so transition would look strange
@nguyensomniac

Reply:
ok, this is fixed